.github/CODEOWNERS

```diff
@@ -480,24 +480,24 @@ static/app/components/events/eventStatisticalDetector/                    @getse
 
 
 ## Issues
+**/issues/**                                        @getsentry/issues
+/src/sentry/api/helpers/source_map_helper.py        @getsentry/issues
+/src/sentry/api/helpers/actionable_items_helper.py  @getsentry/issues
```

Comment:
Unless I make an explicit comment, I'm just re-ordering the list alphabetically.